examples/quickstart/wikiticker-index.json

```diff
@@ -0,0 +1,85 @@
+{
+  "type" : "index_hadoop",
```

Comment:
default ingestion into druid should be to stream it in, once windowperiod is gone it will be possible

hadoop index will be optional